test/tests/functional/pbs_reservations.py

```diff
@@ -85,56 +86,324 @@ def submit_asap_reservation(self, user, jid):
 
         return self.server.submit(r)
 
-    def test_degraded_standing_reservations(self):
+    def degraded_resv_reconfirm(self, start, end, rrule=None, run=False):
         """
-        Verify that degraded standing reservations are reconfirmed on
-        other avaialable vnodes
+        Test that a degraded reservation gets reconfirmed
         """
-
-        a = {'reserve_retry_init': 5, 'reserve_retry_cutoff': 1}
+        a = {'reserve_retry_time': 5}
         self.server.manager(MGR_CMD_SET, SERVER, a)
 
-        a = {'resources_available.ncpus': 4}
+        a = {'resources_available.ncpus': 1}
         self.server.create_vnodes('vn', a, num=2, mom=self.mom)
 
         now = int(time.time())
 
         # submitting 25 seconds from now to allow some of the older testbed
```

Comment:
you might want to change this comment as the start and end time is now passed in by the caller.